project-reports/2023/2023-Q3-Hyperledger-Besu.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: 2023 Q3 Hyperledger Besu
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Hyperledger Besu continues its momentum for usage on public and private Ethereum and Ethereum-like networks. There is an uptick in contributions on the private network side with three contributors being nominated for Maintainer status at Kaleido and Web3Labs. We continue to focus on improvements around Bonsai and readying private networks for the storage format. Client performance is seeing improvement across sync, block processing, and RPC. Discord activity remains heavy, with a relatively even split between public and private network questions and suggestions. Execution clients are becoming an item of note for Ethereum L2s and L3s. Besu is setting strategy around this future and engaging multiple L2 networks, starting with Linea. 
+
+# Questions/Issues for the TOC
+
+Are there remaining asks of Consensys around the CI that should be spelled out?
```

Comment:
The question was discussed in the 28th September 2023 TOC meeting, a staff member will reach out to the maintainers for further discussion.

A note on the similar topic: A template was reviewed to receive the feedback on current CI processes across all the projects. You can find details of it in the same TOC meeting. All maintainers will be sent this questionnaire.